tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+
+To check the Message Processing Log (MPL) of this integration flow from your tenant, go to **Monitoring** > **Overview** > **All Integration flows**. Choose the latest record with your artefact name.
+
+  <!-- border -->![Payload Processed](Payload-Processed.png)
+
+> Ensure to check the time stamp to determine which record to check.
+
+
+
+### Atomic Batch Operation
+
+Atomic Batch Operation doesn't allow an action to be performed on the payload if an error is encountered. To verify that, follow these steps:
+
+
+1. Copy the above payload and modify just one record and make it unique. For example, here we are making the **`emp_id`** (of second record) as 3. Thus, making that record unique, but the first record is duplicate. Send this query.
+
+    ```
+    <root>
+    	<insert_statement1>
+    		<dbTableName action="INSERT">
+    			<table> samplejdbctest</table>
+    			<access>
+    				<emp_id>1</emp_id>
+    				<emp_name hasQuot="Yes">v XXXXX</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement1>
+    	<insert_statement2>
+    		<dbTableName action="INSERT">
+    			<table>samplejdbctest</table>
+    			<access>
+    				<emp_id>3</emp_id>
+    				<emp_name hasQuot="Yes"> g28</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement2>
+    </root>
+    ```
+
+      <!-- border -->![Atomic Batch Operation](Atomic-Operation.png)
+
+     Postman displays error. To get more clarity, let's check the MPL.  
+    <!-- border -->![MPL for Atomic Batch Operation](MPL-Atomic.png)
+
+
+2. Let's confirm, whether unique record (with `emp_id` 3) is inserted in the database or not. To do this, uncheck the **Batch Mode** checkbox and **Deploy** your integration flow.
+
+    <!-- border -->![Batch Mode Off](Batch-Off.png)
+
+Then, **Send** the following query from Postman:
+
+`select * from samplejdbctest`
+
+  <!-- border -->![Atomic Batch Operation Verification](Verify-Atomic.png)
+
+This response shows that no new data has been inserted. Hence, verifying the Atomic operation.
+
+### Non-Atomic Batch Operation
+
+
+Depending upon your driver (which depends upon choice of database), the behaviour of Non- Atomic mode differs. Say for SQL server database, you would be able to insert the unique record and it would fail all the duplicate records. So, the complete payload doesn't fail. But for some other database and driver, it may act similar to Atomic mode and fail the payload even if single error is encountered.
+
+1. In your integration flow, edit the settings for JDBC Receiver adapter. Under **Connections** tab, select the **Batch Payload** checkbox and select **Batch Operation** as Non-Atomic.
+
+    <!-- border -->![Batch Payload Setting Non-Atomic](Batch-Payload-Setting-Non-Atomic.png)
+
+2. **Send** this payload from Postman. Here we are making the **`emp_id`** (of second record) as 4. Thus, making that record unique, but the first record is duplicate.
+
+    ```
```

Comment:
Please add keyword for code highlighting

Reply:
done